x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.test.ts

```diff
@@ -9,7 +9,7 @@ import {
   contextServiceMock,
   executionContextServiceMock,
 } from '@kbn/core/server/mocks';
-import { createHttpServer } from '@kbn/core/server/test_utils';
+import { createHttpServer } from '@kbn/core-http-server-mocks';
```

Comment:
We cannot indefinitely postpone importing directly from packages and the risk seems low enough to start doing that.
@elastic/apm-ui `createHttpServer` was being re-exported from the package in `core/server/test_utils`, so we have no other choice now than to import that directly.